src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -165,6 +165,7 @@ export class MeetingManager implements AudioVideoObserver {
       logLevel,
       postLogConfig,
       videoDownlinkBandwidthPolicy,
+      reconnectTimeoutMs
```

Comment:
nit: Add a trailing comma. I know we have not ran the linting since Harshit added the linting. Could you please run linting once?

I tried to run it as part of the build:release but we have to fix existing linting issues and then add.